### DISCUSSION TITLE
Bookmark sessions

### DISCUSSION THREAD

**Discussion by Jackaed** [olimorris/persisted.nvim#155] [Ideas]
It would be really nice if you could "bookmark" different sessions, and then only search for those favorite sessions through telescope. As it stands, I use persisted with autosave/autoload enabled, which is very handy for when I'm navigating around my system, but there are only a select few projects I actually want to appear when I search through my sessions, rather than seeing the directory of every random place I happen to have opened neovim. 

It would also be nice if you could give these bookmarks a name. 

Would this be possible to implement?

**Comment by olimorris** [comment]
It's not something I have time for but open to a PR to implement this.
